composer.json

```diff
@@ -25,16 +25,17 @@
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.7.1",
-        "phpunit/phpunit": "^6.4"
+        "phpunit/phpunit": "^6.4",
+        "matthiasnoback/symfony-dependency-injection-test": "^2.3"
     },
     "scripts": {
         "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
     },
     "extra": {
         "_ezplatform_branch_for_behat_tests": "master",
         "branch-alias": {
-            "dev-tmp_ci_branch": "2.0.x-dev",
-            "dev-master": "1.0.x-dev"
+            "dev-tmp_ci_branch": "2.1.x-dev",
```

Comment:
This and line below should not be changed. Branch 1.0 should be tested with ezplatform 2.0.x branch.

Reply:
I think the branch is created from master, not 1.0 and that's why it contains 2.1 as branch alias. Github shows some extra commits from master in this branch.